src/Integrations/Integrations/Laminas/LaminasIntegration.php

```diff
@@ -0,0 +1,392 @@
+<?php
+
+namespace DDTrace\Integrations\Laminas;
+
+use DDTrace\Integrations\Integration;
+use DDTrace\SpanData;
+use DDTrace\Tag;
+use DDTrace\Type;
+use DDTrace\Util\Normalizer;
+use Laminas\EventManager\EventInterface;
+use Laminas\Mvc\MvcEvent;
+use Laminas\Router\RouteMatch;
+use Laminas\Stdlib\RequestInterface;
+use Laminas\View\Model\ModelInterface;
+
+use function DDTrace\hook_method;
+use function DDTrace\trace_method;
+
+class LaminasIntegration extends Integration
+{
+    const NAME = 'laminas';
+
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    public function init()
+    {
+        if (!self::shouldLoad(self::NAME)) {
+            return Integration::NOT_LOADED;
+        }
+
+        $rootSpan = \DDTrace\root_span();
+
+        if (is_null($rootSpan)) {
+            return Integration::NOT_LOADED;
+        }
+
+
+        $MVCEvents = [
+            'bootstrap',
+            'dispatch',
+            'dispatch.error',
+            'finish',
+            'render',
+            'render.error',
+            'route'
+        ];
+
+        $events = [
+            'bootstrap',
+            'dispatch',
+            'dispatch.error',
+            'finish',
+            'render',
+            'render.error',
+            'route',
+
+            'mergeConfig',
+            'loadModules',
+            //'loadModule.resolve',
+            //'loadModule',
+            'loadModules.post',
+
+            'renderer',
+            'renderer.post',
+            'response',
+
+            'sendResponse'
+        ];
+
+        $integration = $this;
+
+        hook_method(
+            'Laminas\EventManager\EventManagerInterface',
+            'attach',
+            null,
+            function ($This, $score, $args) use ($MVCEvents) {
+                if (!in_array($args[0], $MVCEvents)) {
+                    return;
+                }
+
+                $listener = $args[1];
+                $className = get_class($listener[0]);
+                $methodName = $listener[1];
```

Comment:
$listener is a callable. Probably all uses you've checked are `[$class, $method]`? But I think this should at least check whether it's actually an array.

(I suppose DDTrace\install_hook should be extended to generally accept any callable, also these written as `["class", "method"]` ...)